tests/memoryview/numpy_memoryview.pyx

```diff
@@ -734,3 +734,18 @@ def same_type_after_arrays_composite():
     cdef SameTypeAfterArraysStructComposite element
     arr = np.ones(2, np.asarray(<SameTypeAfterArraysStructComposite[:1]>&element).dtype)
     cdef SameTypeAfterArraysStructComposite[:] memview = arr
+
+
+ctypedef fused np_numeric_t:
```

Comment:
Could we also try a fused type of `np.float64_t[:]` and `object`. In that case the `dtype="M"` array should match `object` (so not produce any error).

I'm fairly sure it'll work, but it'd be nice to have the explicit test

Reply:
done thx!